project_task_add_very_high/README.rst

```diff
@@ -0,0 +1,87 @@
+.. image:: https://img.shields.io/badge/license-AGPL--3-blue.png
+   :target: https://www.gnu.org/licenses/agpl
+   :alt: License: AGPL-3
+
+==========================
+Project Task Add Very High
+==========================
+
+This module extend the field `priority` in project tasks, adding two new levels of priority.
+The two new levels of priority are: `High` and `Very High`.
+
+
+On a task form, the priority widget is shown as three stars instead of one:
+
+.. figure:: static/description/image.png
+   :alt: On form, priority widget shows three stars instead of one
+
+
+Also on a kanban view, the priority widget is shown as three stars instead of one:
```

Comment:
Suggestion:
"On a Kanban view, the priority widget is displayed as well with three stars instead of one"